src/box/box.cc

```diff
@@ -1469,7 +1469,14 @@ static double
 box_check_replication_synchro_timeout(void)
```

Comment:
Nit: let's fix the commit prefix to `limbo: ...`. You don't touch raft files in this commit

Reply:
Fixed.